charts/pega/README.md

```diff
@@ -801,6 +801,32 @@ pegasearch:
   externalURL: "http://srs-service.namespace.svc.cluster.local"
 ```
 
+To use OAuth authorization service for the connection between Pega Infinity and Search and Reporting Service, follow the instructions provided at ['backingservices'](../backingservices) to configure connection to the authorization service in Search and Reporting Service. You must configure authorization parameters for your Pega Infinity deployment using the parameter in values.yaml as shown in the following table and example:
```

Comment:
can we tweak the wording a little so the context is more clear. Sorry that the original readme wording wasn't that clear...but this should help!

To configure authorization for the connection between Pega Infinity and the Search and Reporting Service (SRS)  use the OAuth authorization service. For more information, see ['backingservices'](../backingservices). To configure the connection to the authorization service in SRS you must configure the following authorization parameters in the Pega values.yaml as shown in the following table and example: